VikingQuiz.UI/src/Components/LoginFormComponent/LoginFormComponent.tsx

```diff
@@ -11,13 +11,13 @@ class LoginFormComponent extends FormComponent {
   public render() {
     return (
       <div className="form-body container-fluid">
-        {this.state.inputs.map((input: InputData) => this.renderInput(input))}
+        {this.state.inputs.map((input: IInputData) => this.renderInput(input))}
         <div className="forget-remember col-xs-12">
-          <a href="#" className="forget-pass col-xs-6">FORGOT PASSWORD</a>
+          <a href="#" className="forget-pass col-xs-6">forgot password</a>
```

Comment:
Please extract all these strings in some constants. Using constants, it will make your life easier when you will need to change the text or add Localization (translated text based on browser language)